FreeRTOS-Plus/ThirdParty/libslirp/.clang-format

```diff
@@ -0,0 +1,58 @@
+# https://clang.llvm.org/docs/ClangFormat.html
```

Comment:
Can libslirp be included as a submodule rather than copied?

Reply:
New commit submitted to submodule libslirp